ui/src/app/cohort-search/redux/actions/service.ts

```diff
@@ -590,6 +661,10 @@ export class CohortSearchActions {
       entities.searchRequests[SR_ID][role] = data[role].map(group => {
         group.items = group.items.map(item => {
           item.searchParameters = item.searchParameters.map(param => {
+            group.mention = group.mention ? group.mention : '';
+            group.time = group.time ? group.time : '';
+            group.timeValue = group.timeValue ? group.timeValue : 0;
+            group.timeFrame = group.timeFrame ? group.timeFrame : '';
```

Comment:
These temporal items should get set in the outer map so they only get set once. Here they are being set for each group item and each item parameter

Reply:
We also need to set `temporalGroup` to 0 for each item if the group is not temporal

Reply:
done